EncounterHandler.js

```diff
@@ -689,7 +689,13 @@ function inject_monster_image(stat) {
 		return;
 	}
 	let url = stat.data.largeAvatarUrl;
-	if (url === undefined || url == null || url.length == 0) {
+	// attempt to get the avatar to
+	let largeAvatarReponse = $.ajax({
+		async: false,
```

Comment:
I'm a little worried that making this synchronous will cause performance issues. If the request takes a long time or times out, the code after this won't execute until the network request completes.